homeassistant/components/vconnex/translations/vi.json

```diff
@@ -0,0 +1,22 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Thiết bị đã được cấu hình"
+        },
+        "error": {
+            "cannot_connect": "Không thể kết nối",
+            "invalid_credentials": "Thông tin đăng nhập không đúng",
+            "credentials_used": "Thông tin đăng nhập đã được sử dụng",
+            "unknown": "Unexpected error"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "endpoint": "Endpoint",
+                    "client_id": "Client ID",
+                    "client_secret": "Client Secret"
+                }
+            }
+        }
+    }
+}
```

Comment:
Only the english file should be added (other langages are generated automatically)
```suggestion
```

Reply:
Removed